app/views/shared/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with model: model, url: url, class: 'form', local: true do |f| %>
+  <% if model == @like %>
+    <%= f.submit 'Like', class: 'btn btn_interact like_btn'%>
+  <% else %>
+    <% fields.each do |field| %>
+      <div class="field">
+        <% case field.to_s %>
+          <% when 'text' %>
+            <%= f.text_area field, placeholder: field.to_s.humanize %>
+          <% else %>
+            <%= f.text_field field, placeholder: field.to_s.humanize %>
+        <% end %>
+      </div>
+    <% end %>
+
+    <div class="actions">
+      <%= f.submit class: 'btn btn_cta btn_cta_new' %>
+    </div>
+  <% end %>
+<% end %>
```

Comment:
- [ ] After fixing the syntax error in the `app/views/posts/new.html.erb` file, I noticed that your form page doesn't display any form input but the _**Like**_ button. The `model` variable is being passed the wrong value. It should be `@posts` instead of `@post`. Take note of the **_s_** that is missing. 💻 🖱️ 

---

![image](https://github.com/Kgomotso196/Blog/assets/77106962/7ef6e448-e1a2-4a33-8a88-5358f8631605)

Reply:
Thank you for your constructive feedback, working on it @Gambit142